sonar-docs/analysis/scan/sonarscanner-for-msbuild.md

```diff
@@ -11,22 +11,19 @@ title: SonarScanner for .NET
 [[info]]
 | **Download SonarScanner for .NET 5.0.0** - [Issue Tracker](https://github.com/SonarSource/sonar-scanner-msbuild/issues) – [Source](https://github.com/SonarSource/sonar-scanner-msbuild)
```

Comment:
Are we clear that it is the version 5 of the scanner for .NET, or is it the version 5 of the scanner for .NET 5 ?

Reply:
This is the version 5 of the Scanner for .NET, coming along with a new flavor, compiled with .NET 5. This will follow the future versions number of .NET (1 new version per year).